ctpolicy/ctpolicy_test.go

```diff
@@ -159,14 +56,10 @@ func TestGetOperatorSCTs(t *testing.T) {
 			groups: loglist.List{
 				"OperA": {
 					"LogA1": {Url: "UrlA1", Key: "KeyA1"},
-					"LogA2": {Url: "UrlA2", Key: "KeyA2"},
```

Comment:
It's not clear to me why some of these extra logs and operators have been removed from the test cases. Yes, they're not strictly necessary because they're not being submitted to, but they are proving "we submit to _only_ two, not every single configured log and operator". Maybe leave them in for just one of the test cases?

Reply:
Ah, this is a mistake on my part. I think made this change while I was doing some troubleshooting with the unit tests. I simply forgot to revert it.